azkaban-common/src/main/java/azkaban/project/Project.java

```diff
@@ -97,7 +99,8 @@ public static Project projectFromObject(final Object object) {
     final int version = (Integer) projectObject.get("version");
     final Map<String, Object> metadata =
         (Map<String, Object>) projectObject.get("metadata");
-
+    final Map<String, Object> featureFlagsMap = (Map<String, Object>) projectObject.get("featureFlags");
+    final Boolean isCRTLocked = (Boolean) projectObject.get("isCRTLocked");
```

Comment:
A class variable with the same name `isCRTLocked` is already defined at line 52; if these two are duplicate then we should remove one; otherwise it is better to rename one of them to avoid confusion

Reply:
I don't quite get why they're duplicate ones.
This variable is used to reconstruct "isCRTLocked" from DB, it could be null as this is newly introduced. and this variable is not class variable but only inside this method, it will be recycled after.

Reply:
ah okay my bad - this is a static method to create an object from JSON (from DB). So it is not related with the class variable